lib/Analysis/Membar.cpp

```diff
@@ -89,6 +104,9 @@ void MembarAnalysis::visitTerminator(Operation *op,
   // Otherwise, it could be a return op
   if (op->hasTrait<OpTrait::ReturnLike>())
     return;
+  // Nothing to be done here.
+  if (isa<triton::gpu::WarpYieldOp>(op))
```

Comment:
Should `WarpYieldOp` be a `ReturnLike` op? `scf.yield` is

Reply:
> `scf.yield` is

I actually find that strange, but yes it could be.